client/src/modules/general-ledger/general-ledger.ctrl.js

```diff
@@ -330,10 +332,27 @@ function GeneralLedgerController(
     });
   }
 
+  // specify if titles accounts should be hidden
+  function toggleHideTitleAccount() {
+    vm.hideTitleAccount = !vm.hideTitleAccount;
+    hideTitles();
+  }
+
+  // Hide when possible title account
+  function hideTitles() {
+    if (vm.hideTitleAccount) {
+      const isNotTitleAccount = (account) => account.type_id !== bhConstants.accounts.TITLE;
```

Comment:
For performance reasons, you should declare this function outside of the `hideTitles()` function.  We don't need to re-create it every single time that we call `hideTitles()`.